resolwe_bio/tests/processes/test_alignment.py

```diff
@@ -307,3 +309,36 @@ def test_subread(self):
         aligned_reads = self.run_process('alignment-subread', inputs)
         self.assertFile(aligned_reads, 'stats', 'subread_paired_reads_report.txt')
         self.assertFile(aligned_reads, 'bam', 'subread_paired.bam')
+
+    @tag_process('alignment-hisat2', 'upload-genome')
+    def test_hisat2_bigwig(self):
+        with self.preparation_stage():
+            paired_reads = self.prepare_paired_reads(mate1=['SRR2124780_1 1k.fastq.gz'],
+                                                     mate2=['SRR2124780_2 1k.fastq.gz'])
+        inputs = {
+            'src': 'hg38_chr21_9M.fa.gz',
+            'species': 'Homo sapiens',
+            'build': 'hg38'
+        }
+        genome_ucsc = self.run_process('upload-genome', inputs)
+        inputs = {
+            'genome': genome_ucsc.id,
+            'reads': paired_reads.id}
+        aligned_reads = self.run_process('alignment-hisat2', inputs)
+        self.assertFile(aligned_reads, 'bigwig', 'hisat2_paired_ucsc_bigwig.bw')
+
+        with self.preparation_stage():
```

Comment:
It's unusual to have two preparation stages in the same test. The rest of the test starting from here does not have any data objects in common with the first part of the test. Consider adding this preparation stage to the first one or split the test into two test methods.